tests/test_bs.py

```diff
@@ -351,7 +351,7 @@ def test_gossip(load_config, setup_clients):
     (out, err) = p.communicate()
     assert '{"value":"ok"}' in out.decode("utf-8")
 
-    gossip_propagation_sleep = len(setup_clients.pods) * 2
+    gossip_propagation_sleep = len(setup_clients.pods)
```

Comment:
comment to explain

Reply:
Initially it was 2 sec delay for each spacemesh node. 
This is only assessment

Reply:
add comment in code to explain this...

Reply:
Done